projects/storefrontapp-e2e-cypress/cypress/e2e/accessibility/aux-key.e2e.cy.ts

```diff
@@ -1,6 +1,14 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { isolateTests } from '../../support/utils/test-isolation';
+
 context('Auxiliary Keys', () => {
   describe('Category Navigation', () => {
-    before(() => {
+    beforeEach(() => {
```

Comment:
rather than flagging it as {TestIsolation=false}, I made this describe block compatible with TestIsolation by changing the hook to run before each 'it' block. tested, no side effects.